packages/documentation/src/elements/navigation-drawer/navigation-drawer.stories.ts

```diff
@@ -0,0 +1,55 @@
+import type { Meta, StoryObj } from "@storybook/web-components";
+import { html } from "lit";
+import { renderIcon } from "../icon/icon.stories.js";
+
+function navigationItem(icon: string, label: string, indicator = 0) {
+  return html`
+    <li>
+      <a class="hap-navigation-drawer-item">
```

Comment:
```suggestion
      <a class="hap-navigation-drawer-item" href="#">
```
Sonst können die Items in der Demo nicht fokussiert werden, um den Fokus-State zu sehen.

Reply:
Ah, mist. ich hatte es extra raus genommen, weil man sonst beim Klick auf eines der Items das Storybook neu lädt, bzw. dann nur noch den Inhalts-IFrame sieht.